docs/changelog.rst

```diff
@@ -10,6 +10,8 @@ Starting with version 1.8.0, pybind11 releases use a `semantic versioning
 IN DEVELOPMENT
 --------------
 
+This is the last version to support Python 2.7 and 3.5.
```

Comment:
Format in **bold**? (Or ALL CAPS if the formatting won't carry through everywhere?)

Reply:
This is already by itself at the top, and I hope it really doesn't affect / matter to anyone anymore. It's not exactly new that a library would be dropping Python 2 at this point.